caesarsCipher/script.js

```diff
@@ -0,0 +1,53 @@
+const form = document.getElementById('my-form');
+let userInput = document.getElementById('user-input');
+let cipherPadding = document.getElementById('cipher-padding');
+let result = document.getElementById('ciphered-word');
+const copyButton = document.getElementById('copy-btn');
+const copied = document.getElementById('copied');
+
+const MIN_CHAR = 65;
+const MAX_CHAR = 90;
+
+const TIMEOUT = 3000;
+
+function cipher(str, padding) {
+    let arr = str.toUpperCase().split('');
+    let newArr = [];
+
+    if (padding === 0) padding = 13;
+
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i].charCodeAt() < MIN_CHAR || arr[i].charCodeAt() > MAX_CHAR) {
+            newArr.push(arr[i]);
+        }
+        else if (arr[i].charCodeAt() + padding > MAX_CHAR) {
+            newArr.push(String.fromCharCode(arr[i].charCodeAt() - padding));
```

Comment:
I would expect to get "A" as the result of this encoding (Z with 1 padding)
![image](https://user-images.githubusercontent.com/13898415/186426365-86dbcbd0-6d82-4cb7-a367-0429a087cb51.png)

Reply:
fixed